crates/holochain/src/conductor/cell.rs

```diff
@@ -1110,12 +914,13 @@ impl Cell {
     #[tracing::instrument(skip(self))]
     pub async fn destroy(self) -> CellResult<()> {
         self.cleanup().await?;
-        let path = self.env.path().clone();
-        // Delete directory
-        self.env
-            .remove()
-            .await
-            .map_err(|e| CellError::Cleanup(e.to_string(), path))?;
+        //TODO "This will need to be reference counted if we really want to delete it"
+        // let path = self.authored_env.path().clone();
+        // // Delete directory
+        // self.authored_env
+        //     .remove()
+        //     .await
+        //     .map_err(|e| CellError::Cleanup(e.to_string(), path))?;
```

Comment:
o_O - would we ever want to delete this?

Reply:
resolved in `5bb543b56142472e18bccba2c54ca17179ad0d14`